onnxruntime/core/providers/op_kernel_type_control.h

```diff
@@ -197,11 +204,12 @@ struct EnabledTypes {
  * @param OpProvider The Op provider.
  * @param OpDomain The Op domain.
  * @param OpName The Op name.
+ * @param OpSet The opset to use for the supported types list.
  * @param ArgDirection Direction of the given Op kernel argument - Input or Output.
  * @param ArgIndex Index of the given Op kernel argument.
  */
 #define ORT_OP_KERNEL_ARG_ENABLED_TYPE_TUPLE(                                       \
-    OpProvider, OpDomain, OpName, ArgDirection, ArgIndex)                           \
+    OpProvider, OpDomain, OpName, Opset, ArgDirection, ArgIndex)                    \
   ::boost::mp11::mp_rename<                                                         \
       ORT_OP_KERNEL_ARG_ENABLED_TYPE_LIST(                                          \
           OpProvider, OpDomain, OpName, ArgDirection, ArgIndex, SupportedTypeList), \
```

Comment:
OpSet should be added to the arguments here too.
or we could remove this macro if there's not a need for it. we can change the other one to return std::tuple if we want to use std types.

Reply:
Will add for now. If we convert all the ops for the next release and don't use this macro we could remove it at that point.